htdocs/js/notebook/notebook_model.js

```diff
@@ -303,6 +303,10 @@ Notebook.create_model = function()
                 select_range(Math.min(start, end), Math.max(start, end));
             }
 
+            _.each(this.cells, function(cell) {
+                cell.deactivate_cell();
+            });
+
```

Comment:
on second thought, reverted 42695cf428b4b9470 in favor of this, it's more explicit what's going on